app/course/directives/courseTable.js

```diff
@@ -318,6 +320,20 @@ courseApp.directive("courseTable", this.courseTable = function ($rootScope, $tim
 					e.preventDefault();
 				}
 			});
+
+			scope.manuallyDeselectAllCourseRows = function() {
```

Comment:
Add comments as to why these functions exist and are called 'manual' as that may not be obvious to a new programmer.